code/datums/ores.dm

```diff
@@ -0,0 +1,195 @@
+/datum/ore
+	/// The type of ore that is sropped. Expected to be a subtype of [/obj/item/stack/ore].
+	var/drop_type
+	/// The lower bound for the amount of randomized ore dropped.
+	var/drop_min = 1
+	/// The upper bound for the amount of randomized ore dropped.
+	var/drop_max = 5
+	/// The probability that the ore will spread to nearby [/turf/simulated/mineral]s when placed.
+	var/spread_chance = 0
+	/// The icon state of the ore used for mining scanner overlays.
+	var/scan_icon_state = ""
+
+/// Called when the containing turf is "mined", such as with a pickaxe or other digging implement.
+/// Returns `TRUE` if the containing turf should be changed to its "dug" state, `FALSE` if it should remain as is.
```

Comment:
Think you could change this to return a set of defines rather than TRUE and FALSE indicating its new state?

Reply:
Done